### PR TITLE
Exposed the passing extra_vars to launch

### PR DESCRIPTION
**Pull request by syncrou**
@bdunne - (ping) Error handling for bad vars will be in the next PR.

### CODE REVIEW

File: lib/ansible_tower_client/job_template.rb [hunk 1]
```diff
@@ -3,8 +3,15 @@ class JobTemplate
     extend CollectionMethods
     include InstanceMethods
 
-    def launch
-      Api.post("#{url}launch/")
+    attr_reader :response
+
+    def launch(vars = {})
+      launch_url = "#{url}launch/"
+      @response = Api.post(launch_url, vars).body
```
Comment (bdunne) [inline lib/ansible_tower_client/job_template.rb]: @syncrou Sorry for the delay, I'm okay with merging this for now, but I feel like this should probably return a Job object.  What do you think?

Comment (syncrou) [inline lib/ansible_tower_client/job_template.rb]: Yes - I like that idea.  I sort of forgot that we could actually build a real class around that response. I'll get that added.

File: lib/ansible_tower_client/job_template.rb [hunk 2]
```diff
@@ -3,8 +3,15 @@ class JobTemplate
...
+    end
+
+    def extra_vars
```
Comment (bdunne) [inline lib/ansible_tower_client/job_template.rb]: Can this be set up on initialize with an `attr_reader` instead?

**Comment by syncrou** [comment]
@bdunne - Just pushed the updates for you to review.

**Comment by bdunne** [comment]
Thanks! :+1:

**Pull request merged**